src/sidebar.jsx

```diff
@@ -243,6 +261,48 @@ const DropdownWithKebab = ({
             title: _("Copy full path")
         },
         { type: "divider" },
+        ...selected.length === 1
+            ? [
+                {
+                    id: "copy-item",
+                    onClick: () => { copyItem(setClipboard, ["/" + path.join("/") + "/" + selected[0].name]) },
```

Comment:
This "/" + path.join("/") + "/" is reused many times .Would worth to be a variable. It is the current directory, no?

Reply:
Fine with followup cleanups.